src/Nethermind/Nethermind.Monitoring/Metrics/MetricsUpdater.cs

```diff
@@ -54,14 +56,23 @@ private void EnsurePropertiesCached(Type type)
         {
             if (!_propertiesCache.ContainsKey(type))
             {
-                _propertiesCache[type] = type.GetProperties().Select(
-                    p => (p, string.Concat(type.Name, ".", p.Name))).ToArray();
+                _propertiesCache[type] = type.GetProperties().Where(p => p.PropertyType != typeof(ConcurrentDictionary<string, long>)).Select(
```

Comment:
Would use .IsAssignableTo and either IDictionary or IEnumerable

Reply:
Added